plugins/@grouparoo/csv/src/initializers/plugin.ts

```diff
@@ -34,13 +34,15 @@ export class Plugins extends Initializer {
       apps: [
         {
           name: "csv",
+          displayName: "CSV",
           options: [],
           methods: { test },
         },
       ],
       connections: [
         {
           name: "csv-import",
```

Comment:
I decided not to change the import name here... what do you think?

Reply:
i'd consider `csv-table-import` or otherwise the same as the other table ones.